packages/wasmd/default.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildGoModule,
+  autoPatchelfHook,
+}:
+buildGoModule rec {
+  pname = "wasmd";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "CosmWasm";
+    repo = "wasmd";
+    rev = "d63bea442bedf5b3055f3821472c7e6cafc3d813";
+    sha256 = "sha256-hN7XJDoZ8El2tvwJnW67abhwg37e1ckFyreytN2AwZ0=";
```

Can you update to the [latest version](https://github.com/CosmWasm/cosmwasm/releases/tag/v1.1.4)?

```suggestion
  version = "1.1.4";

  src = fetchFromGitHub {
    owner = "CosmWasm";
    repo = "wasmd";
    rev = "84bded338d244a29ec7911bb6d003eec7a776caa";
    sha256 = "sha256-ho6T6jBTsHh9j1UAWsj5dERT5EE0APpz5/cm2kR1Hzw=";
```

I got the `sha256` from:

```
nix flake prefetch --json github:CosmWasm/cosmwasm/v1.1.4 | jq '.hash'
```

---

You are right. Just to note, we need the latest version of [wasmd](https://github.com/CosmWasm/wasmd/releases/tag/v0.14.0), not `cosmwasm`